File: WebApplication1/Data/Models/Notification.cs
namespace WebApplication1.Data.Models
{
    public class Notification
    {
        public long ID { get; set; }
        public string Description { get; set; }
        public DateTime DateOfNotification { get; set; }

        public virtual Award Award { get; set; }

        public long AwardId { get; set; }

        public virtual List<Notify> Notifies { get; set; }

        public Notification() { }

    }
}

File: TrueMovieAwards Angular/TrueMovieAwards/src/app/Models/Notification.ts
import { PartialAward } from "./PartialAward"

export class Notification {
    id : number
    description : string
    dateOfNotification : Date
    award : PartialAward

    constructor() {
        this.id = 0
        this.description = ""
        this.dateOfNotification = new Date
        this.award = new PartialAward
    }
}